container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -649,13 +649,21 @@ The SQL function `jseval` allows for the inline definition of functions using Ja
     any SQL expressions and will be bound to the parameters passed in to the
     function.
 
+There are two ways that values in arguments can be represented in Javascript:
+simplified (the default), and a non-simplified representation that is accessed
+by adding a `!` character to the parameter name argument (for example, `!x,y`
+instead of `x,y`).
```

Comment:
That's right... From the doc

> As many argument as are listed in part 2, in the same order. These can be any SQL expressions and will be bound to the parameters passed in to the function.

Then we are back to the alternate function name.